kuzzleio/sdk-javascript
# Documentation v6 and Snippet Tests

## What does this PR do?
Imports the SDK v6 documentation in the `doc/6/` folder and integrates snippet testing.

## How to test?

- `npm run doc-testing` (you may previously want to `docker-compose -f .ci/doc/docker-compose.yml pull`)
- go see Travis builds and check if everything is OK

## Boyscout

@benoitvidis iz teh dude
